src/finder.js

```diff
@@ -109,6 +111,15 @@ export function find (node, args, raw, minimist) {
   }
 }
 
-export function run ({command, args}) {
-  return command.handler.apply(this, args)
+export function run ({command, args}, root) {
+  if (typeof command.handler === 'function') {
+    return command.handler.apply(this, args)
```

Comment:
`.apply(this, args)` is the same logic for every case. It seems you should just decide _how_ to find the command.